activity/scraper.py

```diff
@@ -0,0 +1,95 @@
+import requests
+import json
+import datetime
+from dateutil import parser, relativedelta
+
+# URL to grab all issues from
+issuesurl = "https://coala.github.io/gh-board/issues.json"  # Ignore KeywordBear
```

Comment:
no, dont ignore the bears, they will eat you :P 
derive the `org_name` from your working environment.
See #1 and #26